### DISCUSSION TITLE
How do we swtup admin panel login via the Keycloak as well?

### DISCUSSION THREAD

**Discussion by kdpisda** [urbanplatform/django-keycloak-auth#82] [Q&A]
Right now, after configuring the `django-keycloak-auth`, the admin panel is not working. How do I make it work again?

**Comment by moritz89** [comment]
Check if the error is caused by verify audience. You can disable the check in the Django settings. If that is the case, have a look at: https://www.keycloak.org/docs/latest/server_admin/#_audience_hardcoded

**Reply by moritz89** [reply]
Also, error logs would be helpful :wink:

**Reply by mangeshchavan** [reply]
![Screenshot from 2024-08-31 16-56-16](https://github.com/user-attachments/assets/9db26855-a1b1-4d17-b8ed-6e45ab970264)
